### PR TITLE
Resolving overview graph's not loading

### PR DESCRIPTION
**Pull request by pheinrichs**
This PR resolves signal sensor graphs not loading on health overview page as well, some minor graph changes.

### CODE REVIEW

File: html/includes/graphs/device/cambium_generic_gpsStats.inc.php [hunk 1]
```diff
@@ -19,7 +19,11 @@
     $rrd_options .= ' GPRINT:visible:LAST:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MIN:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MAX:%0.2lf%s\\\l ';
+<<<<<<< HEAD
+    $rrd_options .= " AREA:tracked#00B200:'Tracked       ' ";
+=======
     $rrd_options .= " LINE2:tracked#00ff00:'Tracked       ' ";
+>>>>>>> d0734d78b0d8759e80cd211cdef659e330034682
     $rrd_options .= ' GPRINT:tracked:LAST:%0.2lf%s ';
```
Comment (geordish) [inline html/includes/graphs/device/cambium_generic_gpsStats.inc.php]: Looks like you've missed fixing a merge conflict here.

File: html/includes/graphs/device/cambium_generic_gpsStats.inc.php [hunk 2]
```diff
@@ -19,7 +19,11 @@
     $rrd_options .= ' GPRINT:visible:LAST:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MIN:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MAX:%0.2lf%s\\\l ';
+<<<<<<< HEAD
+    $rrd_options .= " AREA:tracked#00B200:'Tracked       ' ";
+=======
     $rrd_options .= " LINE2:tracked#00ff00:'Tracked       ' ";
+>>>>>>> d0734d78b0d8759e80cd211cdef659e330034682
```
Comment (geordish) [inline html/includes/graphs/device/cambium_generic_gpsStats.inc.php]: Looks like you've missed fixing a merge conflict here.

Comment (geordish) [inline html/includes/graphs/device/cambium_generic_gpsStats.inc.php]: Looks like you've missed fixing a merge conflict here.

Comment (pheinrichs) [inline html/includes/graphs/device/cambium_generic_gpsStats.inc.php]: Ah shoot, I must have missed removing the file. Thanks. It's not doing anything, thought i removed it.

File: html/includes/graphs/device/canopy_generic_gpsStats.inc.php [hunk 3]
```diff
@@ -19,7 +19,11 @@
     $rrd_options .= ' GPRINT:visible:LAST:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MIN:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MAX:%0.2lf%s\\\l ';
+<<<<<<< HEAD
```
Comment (Rosiak) [inline html/includes/graphs/device/canopy_generic_gpsStats.inc.php]: This looks wrong.

**Pull request closed without merging**